onnxruntime/core/session/custom_ops.cc

```diff
@@ -248,6 +248,11 @@ common::Status CreateCustomRegistry(const std::vector<OrtCustomOpDomain*>& op_do
           .SetDomain(domain->domain_)
           .SinceVersion(1);
 
+      auto input_count = op->GetInputTypeCount(op);
+      for (size_t i = 0; i < input_count; i++) {
```

Comment:
Please add unit tests, and documentation for the API changes

Reply:
Will add unit tests and documentation soon.